packages/appstore/src/components/options/stories/options.stories.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import type { Meta, Story } from '@storybook/react';
+//import Options from '../index';
+
+export {};
+// type optionsProps = Parameters<typeof Options>[0];
```

Comment:
why commenting these? if not usable remove them

Reply:
Not using now, removing